src/sources/kubernetes_logs/mod.rs

```diff
@@ -831,4 +1071,187 @@ mod tests {
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "container_id"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "container_image"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "container_name"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "namespace_labels"),
+                Kind::object(Collection::empty().with_unknown(Kind::bytes())).or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "node_labels"),
+                Kind::object(Collection::empty().with_unknown(Kind::bytes())).or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_annotations"),
+                Kind::object(Collection::empty().with_unknown(Kind::bytes())).or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_ip"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_ips"),
+                Kind::array(Collection::empty().with_unknown(Kind::bytes())).or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_labels"),
+                Kind::object(Collection::empty().with_unknown(Kind::bytes())).or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_name"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_namespace"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_node_name"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_owner"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(
+                &owned_value_path!("kubernetes", "pod_uid"),
+                Kind::bytes().or_undefined(),
+                None
+            )
+            .with_event_field(&owned_value_path!("stream"), Kind::bytes(), None)
+            .with_event_field(&owned_value_path!("timestamp"), Kind::timestamp(), None)
```

Comment:
```suggestion
            .with_event_field(&owned_value_path!("timestamp"), Kind::timestamp(), Some("timestamp"))
```

Reply:
Resolved in 3161817cea1c1d85fd5d1050a877208c53db6bb2